integrations/Linux/dashboards/categraf-detail.json

```diff
@@ -898,7 +898,7 @@
                             "id": "byFrameRefID"
                         },
                         "properties": {
-                            "rightYAxisDisplay": "noraml",
```

Comment:
@enkilee he frontend currently depends on `normal`, so we need to keep this as it is for now. Once the frontend implements compatibility, we can update it.